api/integrations/fifaIndexWebScrapper.js

```diff
@@ -1,26 +1,16 @@
+#!/usr/bin/env node
```

Comment:
why is this here?

Reply:
so we can run this from bash with ./fifaIndexWebScrapper.js